zenoh/src/lib.rs

```diff
@@ -272,13 +273,15 @@ where
 
     /// Restrict the matching publications that will be receive by the [`Subscribers`](crate::subscriber::Subscriber)
     /// to the ones that have the given [`Locality`](crate::prelude::Locality).
+    #[cfg(feature = "unstable")]
```

Comment:
`Locality` should not be configurable at session level but only per `declare_subscriber` and `declare_queryable`-basis.